addons/l10n_it_edi/models/account_move.py

```diff
@@ -1239,6 +1239,9 @@ def build_error(message, records):
             if moves := pa_moves.filtered(lambda move: not move.l10n_it_origin_document_type and move.l10n_it_cig and move.l10n_it_cup):
                 message = _("CIG/CUP fields of partner(s) are present, please fill out Origin Document Type field in the Electronic Invoicing tab.")
                 errors['move_missing_origin_document_field'] = build_error(message=message, records=moves)
+        if moves := self.filtered(lambda move: not move.l10n_it_document_type):
+            message = _("please fill out the Document Type field in the Electronic Invoicing tab.")
```

Comment:
maybe this should begin with capital letter (to match other messages). and before merging the sentence should be added to `addons/l10n_it_edi/models/l10n_it_edi.pot`